Exiled.API/Features/Player.cs

```diff
@@ -840,6 +840,13 @@ public int Scp330Usages
         /// </summary>
         internal static ConditionalWeakTable<ReferenceHub, Player> UnverifiedPlayers { get; } = new ConditionalWeakTable<ReferenceHub, Player>();
 
+        /// <summary>
+        /// Gets a <see cref="Player"/> <see cref="IEnumerable{T}"/> filtered by side.
+        /// </summary>
+        /// <param name="side">The player's side.</param>
```

Comment:
Change player's to players' for consistency with other IEnumerable Gets.

Reply:
Sure, I didn't notice it.